### PR TITLE
CONN-4743 | Update ms-service-bus-connector.adoc

### PR DESCRIPTION
**Pull request by alejandrodifiore**
CONN-4743: Need Explanation of Use Cases in the doc

### CODE REVIEW

**Comment by martinpaoloni** [comment]
@dhirsch1 can you please review this? Thanks!

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 1]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
 
+This use case contains flows that create topics, subscriptions, and rules.
+
+* `service-bus-management-demoFlow`
++
+Creates a topic by following these steps:
+
+. Creates the topic description as specified and stores it as the payload.
+. Creates the topic using the stored configuration from the previous step and the topic namespace, obtained from variable `topic.name`.
+. Builds the JSON for the topic.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Converts the newly created topic to a JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 2]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
+. Logs the JSON.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Logs the JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 3]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
++
+* `service-bus-management-demoFlow1`
++
+Creates a subscription by following these steps:
+
+. Creates the subscription description as specified and stores it as the payload.
+. Creates the subscription using the stored configuration from the previous step, the subscription name from the variable `subscription.name`, and the topic name from the variable `topic.name`.
+. Builds the JSON for the subscription.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Converts the newly created subscription to a JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 4]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
+. Logs the JSON.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Logs the JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 5]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
++
+* `service-bus-management-demoFlow2`
++
+Creates a rule by following these steps:
+
+. Creates a rule with the specified configuration and stores it as the payload.
+. Creates the rule using the stored configuration from the previous step, the rule name from the variable `rule.name`, the subscription name from the variable `subscription.name`, and the topic name from the variable `topic.name`.
+. Builds the JSON for the rule.
+. Logs the JSON.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Logs the JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 6]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
...
+. Builds the JSON for the subscription.
+. Logs the JSON.
++
+* `service-bus-management-demoFlow2`
++
+Creates a rule by following these steps:
+
+. Creates a rule with the specified configuration and stores it as the payload.
+. Creates the rule using the stored configuration from the previous step, the rule name from the variable `rule.name`, the subscription name from the variable `subscription.name`, and the topic name from the variable `topic.name`.
+. Builds the JSON for the rule.
```
Comment (martinpaoloni) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```
. Converts the newly created rule to a JSON object.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 7]
```diff
@@ -230,12 +230,14 @@ image::ms-dynamics/ms-service-bus-studio-1.png[General]
 
 == Use Case: Get Queue list
 
+This use case retrieves the list of existing queues in a specific namespace. It returns the list in JSON format.
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```suggestion
This use case retrieves a list of existing queues in a specific namespace. It returns the list in JSON format.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 8]
```diff
@@ -230,12 +230,14 @@ image::ms-dynamics/ms-service-bus-studio-1.png[General]
...
+
 image::ms-dynamics/ms-service-bus-flow.png[General]
 
-. Create a new Mule Application on your Studio and select an HTTP Listener as a Source in the new flow.
-. Add a new HTTP Listener Configuration global element:
+. Create a new Mule Application in Studio and select HTTP Listener as a source in the new flow.
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```suggestion
. Create a new Mule application in Studio and select *HTTP Listener* as a source in the new flow.
```
Is this a physical user selection in a GUI? I know you have some objection but I can't recall why....

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 9]
```diff
@@ -230,12 +230,14 @@ image::ms-dynamics/ms-service-bus-studio-1.png[General]
...
+. Add a new HTTP Listener configuration global element:
 +
-. Specify Host and port parameters with the following values:
+. Specify the Host and Port parameters with the following values:
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc:240]: ditto: code parameters should be ` `; UI labels should be * *.
I will stop marking.

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 10]
```diff
@@ -244,13 +246,13 @@ image::ms-dynamics/ms-service-bus-flow.png[General]
 |Port |8081
 |===
 +
-. Click the Save button.
-. Assign your new  Global configuration to your HTTP Listener.
-. Fill in HTTP Listener path with `/servicebus` value.
-. Drag a new Service Bus component operation in the flow.
+. Click *Save*.
+. Assign the new global configuration to HTTP Listener.
+. Fill in the HTTP Listener path with the value `/servicebus`.
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```suggestion
. Complete the HTTP Listener path with the value `/servicebus`.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 11]
```diff
@@ -259,7 +261,7 @@ output application/json
 payload
 ----
 +
-* Save and run the project as a Mule Application.
+. Save and run the project as a Mule Application.
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]: GLOBAL. No one else init. caps. this.
If you are representing a syntax value or UI label, then....
```suggestion
. Save and run the project as a Mule application.
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 12]
```diff
@@ -375,6 +377,28 @@ For more information about the app log, see xref:mule-runtime::logging-in-mule.a
 
 == Use Case: Windows Service Bus AMQP
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]: Do not use punctuation in topic titles:
```suggestion
== Windows Service Bus AMQP Use Case
```

File: modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc [hunk 13]
```diff
@@ -474,6 +498,35 @@ payload]]></ee:set-payload>
 
 == Use Case: Windows Service Bus Management
 
+This use case contains flows that create topics, subscriptions, and rules.
```
Comment (kbostrom) [inline modules/ROOT/pages/ms-dynamics/ms-service-bus-connector.adoc]:
```suggestion
This use case contains flows that create topics, subscriptions, and rules:
```

**Review by kbostrom — APPROVED** [review]
Edited

**Review by dhirsch1 — APPROVED** [review]

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**